frontend/test/unit/specs/components/ImageDetails/v-copy-license.spec.js

```diff
@@ -20,16 +28,22 @@ describe("VCopyLicense", () => {
         license_url: "http://license.com",
         creator: "John",
         creator_url: "http://creator.com",
+        frontendMediaType: "image",
       },
       fullLicenseName: "LICENSE",
     }
     options = {
+      localVue,
       propsData: props,
+      i18n,
+      pinia: createPinia(),
     }
   })
 
-  it("should contain the correct contents", () => {
-    const wrapper = render(VCopyLicense, options)
-    expect(wrapper.find(".copy-license")).toBeDefined()
+  it("should contain the correct contents", async () => {
+    render(VCopyLicense, options)
+    expect(
+      screen.queryAllByText("media-details.reuse.copy-license.copy-text")
+    ).toHaveLength(3)
```

Comment:
Where is this length coming from? Is there something more specific we can check it against?

Reply:
There are three tabs, so the copy text appears three times. We could check for the presence of the individual tab titles as well if that would be clearer.